src/SocialiteManager.php

```diff
@@ -112,11 +113,11 @@ protected function formatConfig(array $config)
 	/**
 	 * Get the default driver name.
 	 *
-	 * @return string
+	 * @throws InvalidArgumentException
```

Comment:
we still need the leading slash in the docblock, and we need a return string here still since it still defines the return type